posthog/api/action.py

```diff
@@ -190,19 +190,18 @@ def people(self, request: request.Request, *args: Any, **kwargs: Any) -> Respons
         filter = Filter(request=request, team=self.team)
         entity = get_target_entity(filter)
 
-        actors, serialized_actors = TrendsActors(team, entity, filter).get_actors()
+        actors, serialized_actors, raw_count = TrendsActors(team, entity, filter).get_actors()
 
         current_url = request.get_full_path()
         next_url: Optional[str] = request.get_full_path()
+        limit = filter.limit or 100
         offset = filter.offset
-        if len(actors) > 100 and next_url:
```

Comment:
This was basically just an optimistic query earlier: We'd query 200 people, and hope atleast 100 are valid, so we could paginate. Not great.

Got rid of this logic completely & using `filter.limit` to govern this now, just like rest of insight actors.